discord/ext/commands/help.py

```diff
@@ -1341,3 +1342,170 @@ async def send_command_help(self, command):
+            else:
+                cog_name = "Uncategorized Commands"
+            cog_command_mapping.setdefault(cog_name, [])
+            cog_command_mapping[cog_name].append(cmd)
+
+        # Generate the displayable str that will be shown to the user in the embed
+        def sort_uncategorized_first(c):
+            """Sorts the cogs by alphabetical order. Prioritizes uncategorized commands category.
+            
+            Parameters
+            -----------
+            c: :class:`str`
+                The name of the cog.
+            """
+            if c == "Uncategorized Commands":
+                return 0
+            else:
+                return ord(c[0])
+        em_desc = []
+        for cog_name in sorted(list(cog_command_mapping.keys()), key=sort_uncategorized_first):
+            cmd_disp = []
+            for cmd in cog_command_mapping[cog_name]:
+                indent = "\u200b " * 4
+                cmd_disp.append(
+                    f'{indent}__{cmd.name.title()}__{": " + cmd.short_doc if cmd.short_doc != "" else cmd.short_doc}'
+                )
+            cmd_disp = "\n".join(cmd_disp)
+            em_desc.append(f'**{cog_name}:**\n{cmd_disp}')
+        
+        # Generate and send the embed and add the command help data
+        em = discord.Embed(
+            title=f"__**{ctx.bot.user.display_name}'s Command List**__",
+            description=""
+        )
+        em.description = "\n\n".join(em_desc)
+        em.description = f'{em.description}\n\n'
+        em.description = f'{em.description}\nType `{ctx.clean_prefix}help <command>` for more info on a command.'
+        em.description = f'{em.description}\nType `{ctx.clean_prefix}help <category>` for more info on a category.'
+        await ctx.send(embeds=[em])
+
+    async def send_command_help(self, command):
+        # Create the section of the embed showing the description
+        if command.description != "":
+            desc_text = command.description
+        elif command.short_doc != "":
+            desc_text = command.short_doc
+        else:
+            desc_text = "No description provided"
+
+        # Create the section of the embed showing the command usage
+        if command.usage is not None:
+            usage_text = f'Example Usage: `{self.context.prefix}{command.usage}`'
+        else:
+            usage_text = f'Example Usage: `{self.context.prefix}{command.qualified_name} {command.signature}`'
+
+        # Create the section of the embed showing the aliases
+        if command.aliases:
+            alias_text = f'Aliases: `{", ".join(command.aliases)}`'
+        else:
+            alias_text = ""
+
+        # Create and send the embed
+        em = discord.Embed(
+            title=f'__**Command: {command.name.title()}**__',
+            description=f"""
+            **Description:**
+            {desc_text} 
+
+            **More Info:**
+            {usage_text}
+            {alias_text}
+            """
```

Comment:
I feel like triple quoted strings shouldn't be used for this, especially because of the leading whitespace.
```suggestion
            description=f"**Description:**\n"
                               f"{desc_text}\n\n"
                               f"**More Info:**\n"
                               f"{usage_text}\n"
                               f"{alias_text}\n"
```
that screwed up the indentation real good